app/controllers/users/induction_coordinator_registrations_controller.rb

```diff
@@ -1,16 +1,18 @@
 # frozen_string_literal: true
 
-class Users::InductionCoordinators::RegistrationsController < Devise::RegistrationsController
-  def start_registration; end
+class Users::InductionCoordinatorRegistrationsController < Devise::RegistrationsController
+  def start_registration
+    render "users/registrations/induction_coordinators/start_registration"
```

Comment:
I think I'll just change this to users/registrations/start since only induction_coordinators will be registering directly

Reply:
If this becomes Users::RegistrationsController we won't need to specify the view If it's just users/registations/start_registration, right?

Reply:
Yes